client/source/components/questionsAnswers/AnswerForm.jsx

```diff
@@ -26,11 +26,13 @@ const AnswerForm = function CreateAnswerFormComponent({ handlePost }) {
   }, [formInput]);
 
   return (
-    <Form onSubmit={(e) => {
-      e.preventDefault();
-      handlePost(e, formInput);
-      clearForm();
-    }}
+    <Form
```

Comment:
Currently broken. I tried a few variations, and all were rejected with `Error: Question body contains invalid entries`

Example: ```url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/qa/questions'
data: `{"body":"Why did I like this product? WHY DIDN'T I?!\\n\\nLasers, jerpacks, dinosaurs! It has it all! ","name":"MushroomMan13","email":"lordyami@high.kite","product_id":"40444"}````